lib/pyfrc/sim/field/field.py

```diff
@@ -1,80 +1,81 @@
 
+from os.path import abspath, dirname, join
+from tkinter import PhotoImage
+
 import tkinter as tk
 
 from .elements import DrawableElement
 
+
 class RobotField(object):
-    
+
     def __init__(self, root, manager, config_obj):
         '''
-            initializes all default values and creates 
+            initializes all default values and creates
             a board, waits for run() to be called
             to start the board
-            
+
             manager - sim manager class instance
             board_size - a tuple with values (rows, cols)
         '''
-        
+
         # TODO: support drawing an actual field?
-        
+
         self.manager = manager
         self.elements = []      # robots, walls, missles, etc
-        
+
         field_size = config_obj['pyfrc']['field']['w'], \
-                     config_obj['pyfrc']['field']['h']
+            config_obj['pyfrc']['field']['h']
         px_per_ft = config_obj['pyfrc']['field']['px_per_ft']
-        
+
         # setup board characteristics -- cell size is 1ft
         self.cols, self.rows = field_size
         self.margin = 5
         self.cellSize = px_per_ft
-        self.canvasWidth = 2*self.margin + self.cols*self.cellSize
```

Comment:
Isn't this breaking old fields? It would be good for the old field mechanism to work (as they're present in the examples repository).